certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -4692,6 +4757,23 @@
         }
       }
     },
+    "EnvelopeDocumentField": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the Document field.",
+          "type": "string",
+          "x-ms-summary": "name",
```

Comment:
Upper case. This is what shows in the UI
`Name`

Reply:
Done